explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -76,10 +86,11 @@ object ExploreLayout {
   private val component =
     ScalaComponent
       .builder[Props]
-      .stateless
-      .render_P { p =>
-        IfLogged(p.view)((renderFn _).reusable(p))
+      .initialState(State(false))
+      .renderP { ($, p) =>
+        IfLogged(p.view)((renderFn($) _).reusable(p))
```

Comment:
I'm not sure I would recommend this pattern of using `$` to build a reusable function. In this case the state is used in the function, but the state's reusability is not taken into account when computing the reusability of the function.

In other places I've added a `View[State]` as a regular parameter to `renderFn`, and then you can pass a `ViewF.fromState[IO]($)`.